docs/python/python-quick-start.md

```diff
@@ -0,0 +1,125 @@
+---
+Order: 1
+Area: python
+TOCTitle: Quick Start
+ContentId: 77828f36-ae45-4887-b25c-34545edd52d3
+PageTitle: Quick Start Guide for Python in VS Code
+DateApproved: 1/9/2024
+MetaDescription: A quick start guide to get you up and coding with the Python extension in Visual Studio Code.
+MetaSocialImage: images/tutorial/social.png
+---
+
+# Quick Start Guide for Python in VS Code
+
+The Python extension makes VS Code an excellent Python editor, works on any operating system, and is usable with a variety of Python interpreters.
+
+Get started by installing:
+-	[VS Code](https://code.visualstudio.com/)
+-	[A Python Interpreter](https://code.visualstudio.com/docs/python/python-tutorial#_install-a-python-interpreter) (any [actively supported Python version](https://devguide.python.org/#status-of-python-branches))
+-	[Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python) from the VS Code Marketplace
+
+![Gif installing the Python extension in a fresh install of VS Code.](/docs/python/images/quick-start/qs-python-ext-install.gif)
+
+To further customize VS Code for Python, you can leverage the [Python profile template](https://code.visualstudio.com/docs/editor/profiles#_python-profile-template), automatically installing recommended extensions and settings. For Data Scientists, consider using the [Data Science profile template](https://code.visualstudio.com/docs/editor/profiles#_data-science-profile-template).
+
+![Gif showing the Python profile template being installing into VS Code.](/docs/python/images/quick-start/python-profile-create.gif)
+
+## How to create and open a Python project or file
+-	[Open an existing Python project](/docs/python/python-quick-start.md/#open-an-existing-python-project-or-file)
+-	[Create new Python scripts](/docs/python/python-quick-start.md#create-new-python-scripts)
+-	[Clone a Python project from version control](/docs/python/python-quick-start.md#clone-a-python-project-from-version-control)
+
+### Open an existing Python project or file
+
+If you have an existing Python project you wish to work on in VS Code, you can begin by opening your folder or file from the VS Code welcome page or Explorer panel, or by selecting **File -> Open Folder** (Ctrl+K Ctrl+O) or **File -> Open File** (Ctrl+O).
```

Comment:
since it's a quick start maybe we should cover just _one_ flow? 
I think it makes sense to go with the generic File > Open folder since it's applicable for other languages as well.